OWNERS

```diff
@@ -14,7 +14,6 @@ reviewers:
 - chetan-rns
 - mitchharpur
 - keithchong
-- tetchel
 - amitkrout
 - jaideepr97
 - gaganhegde
```

Comment:
Let's remove jaideepr97 and gaganhegde, too.

Reply:
☹️